test/unit/presenters/publishing_api/historical_account_presenter_test.rb

```diff
@@ -102,7 +97,7 @@ class PublishingApi::HistoricalAccountPresenterTest < ActiveSupport::TestCase
 
     historical_accounts_in_descending_order.each_with_index do |historical_account, index|
       links = PublishingApi::HistoricalAccountPresenter.new(historical_account).links
-      actual_names_of_related_people = links[:ordered_related_items].map { |person_hash| person_hash["title"] }
+      actual_names_of_related_people = links[:ordered_related_items].map { |c_id| HistoricalAccount.where(content_id: c_id).first.person.forename }
```

Comment:
It's a bit unecessary to do this via people's names now, since we can just assert on the expected content ids.

E.g:

```
      actual_content_ids_of_related_people = links[:ordered_related_items]
      expected_content_ids_of_related_people = expected_indices_of_surrounding_prime_ministers[index].map do |i|
        historical_accounts_in_descending_order[i].content_id
      end
      assert_equal expected_content_ids_of_related_people, actual_content_ids_of_related_people
```